code/controllers/subsystems/icon_updates.dm

```diff
@@ -0,0 +1,71 @@
+SUBSYSTEM_DEF(icon_update)
+	name = "Icon Updates"
+	wait = 1	// ticks
+	flags = SS_TICKER
+	priority = SS_PRIORITY_ICON_UPDATE
+	init_order = SS_INIT_ICON_UPDATE
+
+	// Linked lists, queue_refs[x] should have TRUE or args stored in queue_args[x]
+	var/list/queue_refs = list()	// Atoms
+	var/list/queue_args = list()	// TRUE or args
+
+/datum/controller/subsystem/icon_update/stat_entry()
+	..("Queue Refs: [queue_refs.len], Queue Args: [queue_args.len]")
```

Comment:
These two lists should always be the same length so just something like `Q:[queue_refs.len]` would be enough.